test/integration/targets/wait_for_connection/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: Test normal connection to target node
+  wait_for_connection:
+    connect_timeout: 5
+    sleep: 1
+    timeout: 10
+
+- name: Test local connection on control master
+  wait_for_connection:
+    connect_timeout: 5
+    sleep: 1
+    timeout: 10
+  connection: local
```

Comment:
I don't really see why that is important. That is the default in VM.
Ironically PEP8 is requiring me to remove the trailing newline, and you are asking me to add one :-)